### PR TITLE
Fix: Headless CMS overview links

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
This PR fix two broken links in the API's overview. Also, this change relates to [this one](https://github.com/vtexdocs/devportal/pull/551) in the `devportal` repo.

#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [x] No, but I am going to.

### CODE REVIEW

File: VTEX - Headless CMS API.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
         "contact": {},
```
Comment (ricardoaerobr) [inline VTEX - Headless CMS API.json:6]: Remover o "contact": {},

**Review by ricardoaerobr — CHANGES_REQUESTED** [review]

**Review by julia-rabello — APPROVED** [review]
LGTM

File: VTEX - Headless CMS API.json (hunk 1 above)
Comment (Mariana-Caetano) [inline VTEX - Headless CMS API.json:6]:
```suggestion
```

Comment (Mariana-Caetano) [reply VTEX - Headless CMS API.json]: Removido, obrigada!

**Review by julia-rabello — APPROVED** [review]

**Review by ricardoaerobr — APPROVED** [review]
LGTM!

**Pull request merged**